bootstrap/src/bootstrap/bootstrap-manager.ts

```diff
@@ -61,8 +56,18 @@ export class BootstrapManager {
       console.log(`ZWED5007I - ${desktops.length} desktops available`);
       console.log('ZWED5008I - desktops: ', desktops);
 
+      const searchParams = new URLSearchParams(window.location.search);
+      const v2Desktop = desktops.filter((desktop)=> { desktop.getIdentifier() == "org.zowe.zlux.ng2desktop" });
+      const v3Desktop = desktops.filter((desktop)=> { desktop.getIdentifier() == "org.zowe.zlux.ivydesktop" });
+      const useV2Desktop = v2Desktop.length == 1 && searchParams.has("use-v2-desktop") && (searchParams.get("use-v2-desktop") == '1'); 
+      
       if (desktops.length == 0) {
         console.error("ZWED5012E - No desktops available to bootstrap.");
+      } else if (useV2Desktop) {
+        console.error("ZWED5324I - The requested Desktop version (V2) is in maintenance mode. To use the newest Desktop version instead, remove the query parameter 'use-v2-desktop'.");
```

Comment:
i should leave a code comment here: i want this to be colored, but its not really an error.
also change to console.warn

Reply:
So like we touched about on Friday's call, there's utility functions for this @1000TurquoisePogs. The window.location.search stuff is not important enough for duplication but here it is if you want better organization:
`getAllApp2AppArgsFromURL( )` in 
zlux-app-manager\virtual-desktop\src\app\start-url-manager\start-url-manager.service.ts

What is relatively important is there's a registration process for stuff like this that should be followed, see:
zlux-app-manager\bootstrap\src\main.ts
`processApp2AppArgs( )`

URL query params get assigned values in window global object and then those get checked for future reference